ZAM_fnc_EZMLiteV2.1.1.sqf

```diff
@@ -6129,11 +6140,15 @@ MAZ_EZM_fnc_initFunction = {
 
 		};
 
+		if(_zeusCanSeeCutscene) then {
+			[1] call MAZ_EZM_fnc_setInterfaceToRefresh;	
```

Comment:
I haven't tested this yet, why use a 1 second delay to refresh the display instead of using the function `MAZ_EZM_fnc_refreshInterface` or just `(findDisplay 312) closeDisplay 0;`? Do those not work as good? 

You could potentially close the Zeus display on each `_zeusPlayer` from the comment above and then reopen it with `openCuratorInterface` after the cinematic completes.

Reply:
Ah, I didn't notice that function, I changed it to use the `MAZ_EZM_fnc_refreshInterface` function instead 👍